simulation/plane.py

```diff
@@ -2,6 +2,9 @@
 import math
 import numpy as np
 
+from typing import List
```

Comment:
deze type hint is outdated. Als je gewoon list[] ipv List[] gebruikt is deze import overbodig